### ISSUE TITLE
WebSeeds

### ISSUE THREAD

**Issue by ieee8023** [bitletorg/bitlet#4]
How should WebSeeds work?

There doesn't seem to be code pulling the url-list out of the metadata. I could write this but then there doesn't seem to be a way to add it to the peers manager. It seems that everything is typed as TorrentPeer and not the superclass Peer.

**Comment by abahgat** [comment]
Yes, you're right: the first step towards implementing this would be making sure that all the code that works with `Peer`s is able to work with the generic interface.

**Comment by disconsented** [comment]
How is this coming along?

**Comment by abahgat** [comment]
Hey @disconsented, we are currently not working on this, but we'd greatly welcome pull requests, if you are interested in making it happen.
